packages/malloy/src/malloy.ts

```diff
@@ -2360,7 +2360,7 @@ export class Runtime {
   public loadSQLBlockByName(
     model: ModelURL | ModelString,
     name: string,
-    options?: ParseOptions & CompileOptions
+    options?: ParseOptions & CompileOptions & PrepareResultOptions
   ): SQLBlockMaterializer {
```

Comment:
Allows `runtime.loadSQLBlockByName(..., { replaceMaterializedReferences: true })`. This is probably not super relevant, as SQL blocks are deprecated in the language.